cmd/zapi/cmd/info/ls.go

```diff
@@ -0,0 +1,68 @@
+package info
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/brimsec/zq/cmd/zapi/cmd"
+	"github.com/brimsec/zq/pkg/colw"
+	"github.com/brimsec/zq/zqd/api"
+	"github.com/mccanne/charm"
+	"github.com/mccanne/charm/pkg/termwidth"
+)
+
+var Ls = &charm.Spec{
+	Name:  "ls",
+	Usage: "ls [-l] [glob1 glob2 ...]",
+	Short: "list spaces or information about a space",
+	Long: `The ls command lists the names and information about spaces known to the system.
+When run with arguments, only the spaces that match the glob-style parameters are shown
+much like the traditional unix ls command.  When used with "-l", each space specified
+is listed with its detailed information as in the info command.`,
+	New: NewLs,
+}
+
+type LsCommand struct {
+	*cmd.Command
+	lflag bool
+}
+
+func NewLs(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
+	c := &LsCommand{Command: parent.(*cmd.Command)}
+	f.BoolVar(&c.lflag, "l", false, "show detail information about each space listed")
+	return c, nil
+}
+
+// Run lists all spaces in the current zqd host or if a parameter
+// is provided (in glob style) lists the info about that space.
+func (c *LsCommand) Run(args []string) error {
+	client := c.Client()
+	matches, err := cmd.SpaceGlob(c.Context(), client, args...)
+	if err != nil {
+		if err == cmd.ErrNoSpacesExist {
+			fmt.Fprintln(os.Stdout, "no spaces exist")
```

Comment:
```suggestion
			fmt.Println("no spaces exist")
```

Reply:
I think in an official cli like this it's good to be specific about where things are headed.

Reply:
fmt.Println is very specific:
```go
func Println(a ...interface{}) (n int, err error) {
	return Fprintln(os.Stdout, a...)
}
```
Please change to use fmt.Println.

Reply:
ok